File: api/v2010/auth_calls_credential_list_mapping/fetch-default/fetch-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$auth_calls_credential_list_mapping = $twilio->sip
                                             ->domains("SDXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                             ->auth
                                             ->calls
                                             ->credentialListMappings("CLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                             ->fetch();

print($auth_calls_credential_list_mapping->friendlyName);

File: api/v2010/auth_calls_credential_list_mapping/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

auth_calls_credential_list_mapping = @client.sip
  .domains('SDXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .auth
  .calls
  .credential_list_mappings('CLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .fetch

puts auth_calls_credential_list_mapping.friendly_name
